types/catchpoint-playwright/.eslintrc.json

```diff
@@ -0,0 +1,72 @@
+{
+    "extends": [
+        "plugin:playwright/recommended"
```

Comment:
We absolutely should not depend on extra plugins in DT, especially not ones that override all of our own eslint configs. Please remove this.